cpp/utils.hpp

```diff
@@ -601,16 +601,13 @@ pack_circumradius_facet(std::shared_ptr<const dolfinx::mesh::Mesh> mesh,
   return {std::move(c), cstride};
 }
 //-----------------------------------------------------------------------------
-xt::xtensor<double, 3>
-get_basis_functions(xt::xtensor<double, 3>& J, xt::xtensor<double, 3>& K,
-                    xt::xtensor<double, 1>& detJ,
-                    const xt::xtensor<double, 2>& x,
-                    xt::xtensor<double, 2> coordinate_dofs,
-                    const std::int32_t index, const std::int32_t perm,
-                    std::shared_ptr<const dolfinx::fem::FiniteElement> element,
-                    const dolfinx::fem::CoordinateElement& cmap)
+void pull_back(xt::xtensor<double, 3>& J, xt::xtensor<double, 3>& K,
```

Comment:
Some documentation of what this function does, what it computes, and what it requires as input and shape of inputs would be nice:D